venue/index.html

```diff
@@ -89,6 +88,7 @@ <h1 class="huge">Venue</h1>
             </div>
         </div>
     </article>
+    <iframe width="100%" height="350" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="http://www.openstreetmap.org/export/embed.html?bbox=-1.5059745311737058%2C52.399833211725685%2C-1.5017688274383545%2C52.40604509180987&amp;layer=mapnik&amp;marker=52.40293598823974%2C-1.5038716793060303" style="border: none"></iframe>
```

Comment:
Shouldn't the iframe still be within the enclosing div (so it remains part of the article)..? For example, could it / should it be on line 89..?